frontend/src/worksheets/tableContents.js

```diff
@@ -62,7 +70,15 @@ export const createHeader = function (worksheet) {
                 bold: true,
               },
               {
-                text: `{partner.BillingAddress}\n`,
+                text: `${country ? country : ''}
```

Comment:
wow ez itt elég kemény, lehet hogy valahol a fájl elején változóba kiszervezve jobb lenne. Nem zavarnak be a sörtörések meg a whitespacek? 🤔

Reply:
jogos, de idő szűke miatt most tekintsünk el tőle pliz.